sbt

```diff
@@ -128,7 +129,10 @@ acquire_sbt_jar () {
     fi
   fi
   local user_home && user_home=$(findProperty user.home)
-  download_jar="${user_home:-$HOME}/.cache/sbt/boot/sbt-launch/$launcher_sv/sbt-launch-$launcher_sv.jar"
+  local download_jar="${user_home:-$HOME}/.cache/sbt/boot/sbt-launch/$launcher_sv/sbt-launch-$launcher_sv.jar"
+  if [[ -d "$sbt_cache" ]]; then
+    download_jar="${sbt_cache}/boot/sbt-launch/$launcher_sv/sbt-launch-$launcher_sv.jar"
+  fi
```

Comment:
Could we define a Bash function called `cache_dir ()` or something that encapsulates this logic here plz?

Reply:
This whole different download/cache wasn't needed, these are always `~/.sbt/boot`, so its just the setting that needed to be updated.

Reply:
oh snap, technically these are suppose to be `--sbt-boot` maybe lol, different issue though.